app/backend/controllers/login-signup.js

```diff
@@ -86,6 +102,31 @@ module.exports.login = async (request, response) => {
   }
 }
 
+module.exports.google = async (request, response) => {
```

Comment:
User's Google ID isn't a secret between our backend and Google. Any other system may retrieve that info the way we do. We should make sure that signing the user in with only that info suffices from a security perspective. We might deny CORS requests or implement some other countermeasure to conform our data integrity.